packages/lib/modules/transactions/transaction-steps/lib.tsx

```diff
@@ -133,6 +133,9 @@ type TransactionInput =
   | ManagedErc20TransactionInput
   | ManagedSendTransactionInput
 
+// In some cases, the onSuccess callback must be retried, for instance, when the refetchApprovals has not finished yet
+export const RetryOnSuccess = 'retry'
```

Comment:
with that name it seems like it will retry if success happens, wouldn't retry be enough?

Reply:
👍 Renamed to `Retry`